resources/kcp/charts/kyma-environment-broker/templates/api-rule.yaml

```diff
@@ -21,3 +21,15 @@ spec:
         - handler: oauth2_introspection
           config:
             required_scope: ["cld:read"]
+    - path: /upgrade/.*
```

Comment:
do we have a path`/upgrade` ?

Reply:
yes, within upgrade API we defined:

- `POST /upgrade/kyma` but in the future we'll also support `/upgrade/cluster`
- `GET /orchestratiions/:id`